src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixignore

```diff
@@ -88,6 +88,8 @@ Microsoft.VisualStudio.ProjectSystem.Query.resources.dll
 Microsoft.VisualStudio.Services.TestManagement.TestPlanning.WebApi.dll
 Microsoft.VisualStudio.Services.TestResults.WebApi.dll
 Microsoft.VisualStudio.Setup.Common.dll
+Microsoft.Web.XmlTransform.dll
+Microsoft.Web.XmlTransform.resources.dll
```

Comment:
When I look at my local VS install, I see Microsoft.Web.XmlTransform.dll in NuGet's install directory, and a subdirectory under MSBuild.  So, I feel reasonably confident that removing it from the vsix will cause features that need this dll to fail.

I'm not sure if we have Apex or VS E2E tests that test the functionality that uses this dll, but even if we do since our VS tests aren't currently working, we need to be careful about regressing VS features.

Reply:
I must have had some transient error saying it couldn't find these assemblies, as it seems to be fine now. Undid the change.